.changeset/few-snakes-travel.md

```diff
@@ -0,0 +1,11 @@
+---
+"@fluidframework/container-definitions": minor
+"__section": deprecation
+---
+Deprecated "policies" and "downloadSummary" on "IContainerStorageService"
```

Comment:
Should the announcement titles be more along the lines of "Introduced new interface X to replace Y between layers A and B"? And in the details note the properties that are deprecated because they don't apply between those layers.